### PR TITLE
feat: Make it possible to provide certificates as a file path or as a raw content

### PR DESCRIPTION
**Pull request by mykola-mokhnach**

### CODE REVIEW

File: lib/subcommands/keychain.js [hunk 1]
```diff
@@ -1,35 +1,83 @@
...
-  await this.exec('keychain', {
+commands.addRootCertificate = async function addRootCertificate (cert, opts = {}) {
+  const {
+    raw = false,
+  } = opts;
+  const execMethod = async (certPath) => await this.exec('keychain', {
     args: [this.requireUdid('keychain add-root-cert'), 'add-root-cert', certPath],
   });
+  if (raw) {
+    await handlePayloadFromFile(cert, execMethod);
```
Comment (imurchie) [inline lib/subcommands/keychain.js]: This function name makes it seem to do the opposite of what it does.

Comment (mykola-mokhnach) [reply lib/subcommands/keychain.js]: Yep, I'm out of imagination for today :(
any better proposals for its name?

File: lib/subcommands/keychain.js [hunk 2]
```diff
@@ -1,35 +1,83 @@
...
-  await this.exec('keychain', {
+commands.addCertificate = async function addCertificate (cert, opts = {}) {
+  const {
+    raw = false,
+  } = opts;
+  const execMethod = async (certPath) => await this.exec('keychain', {
     args: [this.requireUdid('keychain add-cert'), 'add-cert', certPath],
   });
+  if (raw) {
+    await handleRawPayload(cert, execMethod);
```
Comment (imurchie) [inline lib/subcommands/keychain.js]: Beat me to answering your question. :)

But also, is there a reason that the `execMethod` function needs to be called as a callback? Does it not work to do something like
```js
let certPath = cert;
if (raw) {
  certPath = await handleRawPayload(cert);
}
await execMethod(certPath);
```

Comment (mykola-mokhnach) [reply lib/subcommands/keychain.js]: no, because I would like to make sure the temporary folder is deleted after the whole block is executed

Comment (imurchie) [reply lib/subcommands/keychain.js]: Makes sense. Thanks!

**Pull request merged**